.github/workflows/ci-workflow.yml

```diff
@@ -43,7 +43,6 @@ jobs:
           bundler-cache: true
       - run: bundle exec rubocop --format github
       - run: bundle exec haml-lint
```

Comment:
There's a similar job to do at some point for `haml-lint`. Added a card to the design system board for this.